_data/guides.yaml

```diff
@@ -31,6 +31,9 @@ categories:
       - title: Securing WildFly Apps with SAML on OpenShift
         url: /guides/security-saml-openshift
         description: Learn how to secure applications deployed to WildFly on OpenShift with SAML.
+        title: Securing WildFly Apps with Okta on OpenShift
```

Comment:
This is missing the `-` before the title.

Reply:
fixed!